docs/specs/servers/relay/relay-server-rpc.md

```diff
@@ -424,6 +424,32 @@ Used to unregister an active watch webhook corresponding to a webhookId.
 }
 ```
 
+### Batch Request
+
+Allows combining multiple requests into a single one. Used to save on round-trip latency.
+
+```jsonc
+// Request (client->server)
+{
+  "id" : "1",
+  "jsonrpc": "2.0",
+  "method": "irn_batchRequest",
+  "params" : {
+    "batch": Request[],
+    "mode": "sequence" | "concurrent"
```

Comment:
I thought we had a use-case where we need both `sequence` and `concurrent` at the same time. 
Like:
- execute 3 requests concurrently
- afterwards execute another request

Isn't it the case?

Reply:
Correct. So with the suggested implementation you'd combine requests like this:
```js
{
  "method": "irn_batchRequest",
  "params": {
    "mode": "sequence",
    "batch": [
      // First execute a batch of requests concurrently.
      {
        "method": "irn_batchRequest",
        "params": {
          "mode": "concurrent",
          "batch": [
            // Concurrent requests, e.g. subscriptions.
          ]
        },
        ...
      },

      // Then another request once the first batch is processed.
      {
        "method": "irn_publish",
        ...
      },
    ]
  },
  ...
}
```